Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.11
+ADD . /usr/src/nimbess-agent
+RUN mkdir -p $GOPATH/src/github.com/nimbess/nimbess-agent
+WORKDIR $GOPATH/src/github.com/nimbess/nimbess-agent
+COPY . .
+RUN ./build/build-go.sh
```

Comment:
Curious why we use a build.sh script rather than just adding the line directly in the Dockerfile?

Reply:
It's cleaner and it allows me to test building outside of using docker.